__init__.py

```diff
@@ -16,21 +19,11 @@ def __init__(self):
         self.connected = False
         self.wifi_process = None
         self.debug = False  # dev setting, VERY VERBOSE DIALOGS
-        # TODO expose all this in skill settings
-        self.ssid = "OVOS"
-        self.pswd = None
+        self.setup_url = None
         self.grace_period = 45
         self.time_between_checks = 30  # seconds
         self.mycroft_ready = False
-        self.wifi_command = "sudo /usr/local/sbin/wifi-connect --portal-ssid {ssid}"
-        if self.pswd:
-            self.wifi_command += " --portal-passphrase {pswd}"
-        if "color" not in self.settings:
-            self.settings["color"] = "#FF0000"
-        if "stop_on_internet" not in self.settings:
-            self.settings["stop_on_internet"] = False
-        if "timeout_after_internet" not in self.settings:
-            self.settings["timeout_after_internet"] = 90
+        self.wifi_command = "sudo wifi-connect"
```

Comment:
Can we leave a TODO line here to come back to it later? Rather at some point want to remove all "sudo" from the code.

Reply:
it can already be set via skill settings with this PR

Reply:
Ah, right! Just saw line #36 indeed. Nevermind then.

Reply:
I'm not sure why `sudo` is required here; I thought being in `sbin` was all you needed to do to ensure it elevates to root perms